Chummer/sheets/Notes set.xslt

```diff
@@ -238,6 +215,191 @@
+                <xsl:sort select="fullname"/>
+                <xsl:call-template name="weaponnotes"/>
+              </xsl:for-each>
+            </table>
+          </div>
+          <xsl:call-template name="RowSummary">
+            <xsl:with-param name="text" select="$lang.MeleeWeapons"/>
+            <xsl:with-param name="blockname" select="'MeleeWeaponsBlock'"/>
+          </xsl:call-template>
+        </xsl:if>
+
+        <xsl:variable name="gearlist" select="gears/gear[iscommlink != 'True' and (notes != '' or children/gear/notes != '')]"/>
+        <xsl:if test="$gearlist">
+          <div class="block" id="GearBlock">
+            <table><tr><td/></tr></table>
+            <xsl:call-template name="TableTitle">
+              <xsl:with-param name="name" select="$lang.Gear"/>
+            </xsl:call-template>
+            <table class="tablestyle">
+              <tr>
+                <th width="90%" style="text-align: left">
+                  <xsl:value-of select="$lang.Gear"/>
+                </th>
+                <th width="10%"/>
+              </tr>
+              <xsl:for-each select="gears/gear[iscommlink != 'True' and (notes != '' or children/gear/notes != '')]">
+<!--
+                <xsl:sort select="location"/>
+-->
+                <xsl:sort select="name"/>
+                <xsl:call-template name="gearnotes"/>
+              </xsl:for-each>
+            </table>
+          </div>
+          <xsl:call-template name="RowSummary">
+            <xsl:with-param name="text" select="$lang.Gear"/>
+            <xsl:with-param name="blockname" select="'GearBlock'"/>
+          </xsl:call-template>
+        </xsl:if>
+
+        <xsl:variable name="devicelist" select="gears/gear[iscommlink = 'True' and (notes != '' or children/gear/notes != '')]
+             or armors/armor/gears/gear[iscommlink = 'True' and (notes != '' or children/gear/notes != '')]
+             or cyberwares/cyberware/gears/gear[iscommlink = 'True' and (notes != '' or children/gear/notes != '')]
+             or cyberwares/cyberware/children/cyberware/gears/gear[iscommlink = 'True' and (notes != '' or children/gear/notes != '')]
+             or weapons/weapon/accessories/accessory/gears/gear[iscommlink = 'True' and (notes != '' or children/gear/notes != '')]"/>
+        <xsl:variable name="programlist" select="gears/gear[isprogram = 'True' and (notes != '' or children/gear/notes != '')]
+             or armors/armor/gears/gear[isprogram = 'True' and (notes != '' or children/gear/notes != '')]
+             or cyberwares/cyberware/gears/gear[isprogram = 'True' and (notes != '' or children/gear/notes != '')]
+             or cyberwares/cyberware/children/cyberwear/gears/gear[isprogram = 'True' and (notes != '' or children/gear/notes != '')]
+             or weapons/weapon/accessories/accessory/gears/gear[isprogram = 'True' and (notes != '' or children/gear/notes != '')]"/>
```

Comment:
See my comment about nesting gear and nesting gear parents in `Shadowrun 5 set.xslt`.